internal/service/sagemaker/sagemaker_test.go

```diff
@@ -81,8 +102,8 @@ func TestAccSageMaker_serial(t *testing.T) {
 		},
 		"Space": {
 			acctest.CtBasic:            testAccSpace_basic,
-			acctest.CtDisappears:       testAccSpace_tags,
-			"tags":                     testAccSpace_disappears,
+			acctest.CtDisappears:       testAccSpace_disappears,
```

Comment:
is this relevant to the PR?

Reply:
No, I saw a typo where functions are mismatched - tags pointing to disappears. Have reverted considering outside scope this PR